src/adapters/pois_styles.js

```diff
@@ -0,0 +1,10 @@
+
+export const filteredPoisStyle = {
+  type: 'symbol',
+  layout: {
+    'icon-image': 'pin_with_dot',
+    'icon-allow-overlap': true,
+    'icon-size': 0.5,
+    'icon-anchor': 'bottom',
```

Comment:
(Very minor)

Given the shape of the icon this anchor is the right thing to do :+1: 
Note that it was not the case before, so this makes the popup position a bit off:

|Before|After|
|-|-|
|![image](https://user-images.githubusercontent.com/4726554/93086012-bf1b8300-f696-11ea-872d-325d2913bcf3.png)|![image](https://user-images.githubusercontent.com/4726554/93086284-2e917280-f697-11ea-95ee-17c1f8b96767.png)|

Reply:
Ok, I've changed it globally in the Popup util for now, it would be too much refacto here to adapt to all cases. I put a value that ensures the popup doesn't overlap with these hovered marker while not being too far for the "normal" POIs.